prow/github/types.go

```diff
@@ -68,6 +68,18 @@ const (
 	MergeSquash PullRequestMergeType = "squash"
 )
 
+func unmarshalClientError(b []byte) error {
+	clientError := ClientError{}
+	if err := json.Unmarshal(b, &clientError); err != nil {
+		alternativeClientError := AlternativeClientError{}
+		if err := json.Unmarshal(b, &alternativeClientError); err != nil {
```

Comment:
nit: If you invert these conditionals you can reduce the nested indentation and make the code a little easer to read.